public/app/core/utils/explore.ts

```diff
@@ -374,10 +380,11 @@ const parseRawTime = (value: any): TimeFragment | null => {
     return toUtc(value, 'YYYY-MM-DD HH:mm:ss');
   }
 
-  if (!isNaN(value)) {
-    const epoch = parseInt(value, 10);
-    return toUtc(epoch);
-  }
+  // [Zoltan] There shouldn't be a number here
```

Comment:
I see in this case why this was removed. Probably makes sense, as we type this better if we put number here hopefully it should trigger compile time errors somewhere

Reply:
yep. so what now? should I remove this? or uncomment and add back any?

Reply:
removed it